src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -69,6 +86,28 @@ export default class ReplyField extends React.Component<Props, State> {
         this.saveCursorPosition();
     }
 
+    getItemList = (editorState: EditorState): Collaborator[] => {
+        const { collaborators } = this.props;
+
+        const activeMention = getActiveMentionForEditorState(editorState);
+        if (!activeMention) {
+            return [];
+        }
+
+        const trimmedQuery = activeMention.mentionString.trim();
+
+        if (trimmedQuery.length < 2) {
```

Comment:
What is the significance of the number 2 here?

Reply:
> What is the significance of the number 2 here?

Will add comment: "fuzzySearch doesn't match anything if query length is less than 2. Compared to empty list, full list has a better user experience"